zookeeper-server/src/main/java/org/apache/zookeeper/cli/RestoreCommand.java

```diff
@@ -168,9 +178,30 @@ public CliCommand parse(String[] cmdArgs) throws CliParseException {
   @Override
   public boolean exec() throws CliException {
     if (cl.hasOption(OptionShortForm.HELP)) {
-      System.out.println(getUsageStr());
+      out.println(getUsageStr());
       return true;
     }
+    if (cl.hasOption(OptionShortForm.OVERWRITE)) {
+      Scanner scanner = new Scanner(System.in);
+      boolean repeat = true;
+      while (repeat) {
+        out.println(
+            "Are you sure you want to overwrite the destination directories? Please enter \"yes/no\".");
+        String input = scanner.nextLine().toLowerCase();
+        switch (input) {
+          case "yes":
+            repeat = false;
+            break;
+          case "no":
+            out.println(
+                "Exiting restoration. Please remove the \"overwrite\" option from the command, and try again.");
+            return true;
```

Comment:
Should this be `false` since we aren't restoring it? Or should it be true?

Reply:
I returned `true` here because the command finishes without an error. But I think your suggestion makes more sense, we should return `false` because no restoration is done. Will change it.